README.md

```diff
@@ -51,27 +51,27 @@ class app_widgetContainer {
 }
 ```
 
-For methods, follow the standard Apple convention of referring to the first parameter in the method name:
+For methods, when the first argument is part of a prepositional phrase, make the preposition the first label and use the actual argument name inside the method definition:
 
 ```swift
 class Guideline {
-  func combineWithString(incoming: String, options: Dictionary?) { ... }
-  func upvoteBy(amount: Int) { ... }
+  func combine(with incoming: String, options: Dictionary?) { ... }
+  func upvote(by amount: Int) { ... }
```

Comment:
Agreed. Sometimes this is useful, but it's a rare case. Usually, the context is clear at the call site. It doesn't need to be a hard rule.